### DISCUSSION TITLE
Support for Rockerfile

### DISCUSSION THREAD

**Discussion by ns-gdadi** [renovatebot/renovate#31566] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

github

### Please tell us more about your question or problem

We want Renovate to update versions in Rockerfiles. I read the documentation and trying to do this:
`{"regexManagers": [{"fileMatch": ["^Rockerfile$"],  // Ensure it matches the Rockerfile name"matchStrings": ["FROM\\s+(?<depName>\\S+):(?<currentValue>\\S+)"  // Regex pattern to match FROM statements],"datasourceTemplate": "docker",  // Use docker as the data source"versioningTemplate": "docker"}]}`
Are there any plans for supporting Rockerfiles in the future?
cc: @ns-svemu

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
If there were plans, you'd find them in this repo
